doc/connectivity/bluetooth/api/mesh/sar_cfg_srv.rst

```diff
@@ -0,0 +1,105 @@
+
+   (:kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_STEP` + 1) * 25 ms
+
+* SAR Unicast Retransmissions Interval Increment: A value that controls the interval increment used for delaying the retransmissions of unacknowledged segments of a segmented message to a unicast address.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_INC` to set a value.
+  The increment is measured in milliseconds and calculated using the following formula::
+
+   (:kconfig:option:`CONFIG_BT_MESH_SAR_TX_UNICAST_RETRANS_INT_INC` + 1) * 25 ms
+
+* SAR Multicast Retransmissions Count: A value that controls the total number of retransmissions of a segmented message to a multicast address.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_COUNT` to set the value.
+
+* SAR Multicast Retransmissions Interval Step: A value that controls the interval between retransmissions of all segments in a segmented message to a multicast address.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_INT` to set a value.
+  The interval is measured in milliseconds and calculated using the following formula::
+
+   (:kconfig:option:`CONFIG_BT_MESH_SAR_TX_MULTICAST_RETRANS_INT` + 1) * 25 ms
+
+The SAR Receiver state is a composite state that controls the number and timing of Segment Acknowledgment transmissions and the discarding of reassembly of a segmented message.
+It includes the following states:
+
+* SAR Segments Threshold: A value that defines a threshold in number of segments of a segmented message for acknowledgment retransmissions.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_RX_SEG_THRESHOLD` to set the value.
+  When the number of segments of a segmented message is above this threshold, the stack will additionally retransmit every acknowledgment message the number of times given by the value of :kconfig:option:`CONFIG_BT_MESH_SAR_RX_ACK_RETRANS_COUNT`.
+
+* SAR Discard Timeout: This value defines the time since the last successfully received segment before giving up the reception of a segmented message.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_RX_DISCARD_TIMEOUT` to set the value.
+
+* SAR Acknowledgment Delay Increment: This value controls the delay increment of an interval used for delaying the transmission of an acknowledgment message after receiving a new segment.
+  Use the Kconfig option :kconfig:option:`CONFIG_BT_MESH_SAR_RX_ACK_DELAY_INC` to set a value.
+  The increment is measured in segments and calculated using the following formula::
+
+   :kconfig:option:`CONFIG_BT_MESH_SAR_RX_ACK_DELAY_INC` + 1.5
+
+* SAR Acknowledgment Retransmissions Count: This value defines the total number of retranmissions of an acknowledgment message that the stack will additionally send when the size of segments in a segmented message is above the :kconfig:option:`CONFIG_BT_MESH_SAR_RX_SEG_THRESHOLD` value.
```

Comment:
This configuration is missing its :kconfig:option: (see comment above about first mention)